scala-package/examples/src/main/java/org/apache/mxnetexamples/infer/javapi/objectdetector/SSDClassifierExample.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnetexamples.infer.javapi.objectdetector;
+
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.mxnet.javaapi.*;
+import org.apache.mxnet.infer.javaapi.ObjectDetector;
+
+// scalastyle:off
+import java.awt.image.BufferedImage;
+// scalastyle:on
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+import java.io.File;
+
+public class SSDClassifierExample {
+	@Option(name = "--model-path-prefix", usage = "input model directory and prefix of the model")
+	private String modelPathPrefix = "/model/ssd_resnet50_512";
+	@Option(name = "--input-image", usage = "the input image")
+	private String inputImagePath = "/images/dog.jpg";
+	@Option(name = "--input-dir", usage = "the input batch of images directory")
+	private String inputImageDir = "/images/";
+	
+	final static Logger logger = LoggerFactory.getLogger(SSDClassifierExample.class);
+	
+	static List<List<ImmutablePair<String, List<Float>>>>
+	runObjectDetectionSingle(String modelPathPrefix, String inputImagePath, List<Context> context) {
+		Shape inputShape = new Shape(new int[] {1, 3, 512, 512});
+		List<DataDesc> inputDescriptors = new ArrayList<DataDesc>();
```

Comment:
Because of the line limits imposed by ScalaStyle this still spans 2 lines. Because of this my preference is to leave as 2 commands because I think it's slightly more readable than a single command spanning 2 lines. I don't have strong feelings on this though.